server/app/middlewares/websocket_backend.rb

```diff
@@ -86,7 +86,7 @@ def on_open(ws, req)
       end
 
       logger.info "node opened connection: #{node.name || node_id}, labels: #{labels}"
-      node_plugger.plugin!
+      EM.defer { node_plugger.plugin! }
```

Comment:
Should the `on_close` -> `Agent::NodeUnplugger.unplug!` also be moved into a defer thread?